geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestCommand.scala

```diff
@@ -79,6 +79,9 @@ object IngestCommand {
       "to override file extension recognition")
     var format: String = null
 
+    @Parameter(names = Array("-skH", "--skip-header"), description = "flag to skip the first line (header) of a csv/tsv data file")
```

Comment:
I was leaning towards no short version.  All other options have a short version, but I think this is a little difference since it's a flag.